pkg/options/target.go

```diff
@@ -0,0 +1,17 @@
+package options
+
+import (
+	"github.com/lilypad-tech/lilypad/pkg/data"
+	"github.com/spf13/cobra"
+)
+
+func AddTargetCliflags(cmd *cobra.Command, targetConfig *data.TargetConfig) {
```

Comment:
```suggestion
func AddTargetCliFlags(cmd *cobra.Command, targetConfig *data.TargetConfig) {
```
nit: Name should capitalize the `F` in flags.